pkg/extension/sumologicextension/extension.go

```diff
@@ -44,6 +45,7 @@ const (
 	heartbeatUrl             = "/api/v1/collector/heartbeat"
 	registerUrl              = "/api/v1/collector/register"
 	DefaultHeartbeatInterval = 15 * time.Second
+	collectorCredentialsPath = "/tmp/sumo_otc_creds"
```

Comment:
I think this path is wrong. It's temporary and platform dependent. Shouldn't we device a common directory where such information is stored? Perhaps do as IC does?

Reply:
I concur. I'm not sure what IC does but home dir makes sense. Create `~/.sumologic-otel-collector/` and store credential files there?